Fundamentals of web development/main.css

```diff
@@ -0,0 +1,120 @@
+body {
+    font-family: 'Montserrat', sans-serif;
+    background-image: url('images/background.svg');
+    background-size: cover; 
+    background-position: center; 
+    background-repeat: no-repeat; 
+    color: white;
+    margin:0;
+}
+
+nav {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin: 20px;
+    ul {
+        transform: translateX(100%);
+        transition: transform 0.5s ease-in-out;
+        position: fixed;
+        height: 100vh;
+        width:40%;
+        text-align: right;
+        right:0;
+        top:0;
+        list-style: none;
+        li {
+            margin: 0.8em;
+            a{
+            text-decoration-style: none;
+            color: white;
+            }
+        }
+    hamburger img {
```

Comment:
I think on the design its outline in almost a button here. You could achieve this by adding a border and setting width / height.